docs/02-membership.md

```diff
@@ -44,7 +44,7 @@ The membership is a set of people working within the eligible projects who have
 | [Piper Merriam](https://github.com/pipermerriam/) | 1 | Portal Network (EF) | |
 | [Nick Gheorghita](https://github.com/njgheorghita) | 1 | Portal Network (EF) | |
 | [Guru](https://github.com/gurukamath/) | 1 | | [ethereum/execution-specs](https://github.com/ethereum/execution-specs) |
-| [Mário Havel](https://github.com/taxmeifyoucan) | 1 | Protocol Support (EF) | |
+| [Mário Havel](https://github.com/taxmeifyoucan) | 1 | Protocol Support (EF) | [Protocol Fellowship](https://github.com/eth-protocol-fellows) |
```

Comment:
EPF isn't [currently eligible](https://protocol-guild.readthedocs.io/en/latest/01-eligibility.html), though it was mentioned in the [original member nomination](https://github.com/protocolguild/documentation/pull/45). that inconsistency is on me for including it in the nomination, before we had stronger norms about only including eligible work

Is there other work from the original nomination that you're still focused on?

Reply:
Thanks Trent, I wanted to link a single thing rather than couple of smaller ones. Most of the original contributions are still relevant, I can list some of those instead - ephemery, crawler, verkle preimages, pg membership, deb client packages..
Also I think we could discuss the criteria for EPF kind of work with the guild. Or just the situation where I am basically working with small teams and contributors to many core related projects.

Reply:
This is a tricky problem across PG, and I think it's good to show the list as a WIP attempt to clarify things, but these update PRs are almost certainly the wrong "forum" to make decisions about membership eligibility. For example, most of the researchers in PG can't quite map their work to a single repo (or at least one that's already listed) as "Consensus work", "Cryptography", and "Mechanism design" don't have a single canonical repo.

Reply:
coming back to this, PF should be removed and replaced by eligible work that you were nominated for